src/TodoListItem.jsx

```diff
@@ -1,6 +1,6 @@
-const TodoListItem = ({todo}) => {
+const TodoListItem = ({id, todo}) => {
     return (
-        <li>{todo}</li>
+        <li key={id}>{todo}</li>
```

Comment:
Since this component's return statement is only returning a single element, the `key` is not needed. `key`s are only useful when rendering a bunch of sibling components like we do with this component in the TodoList component.